src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +89,13 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            onClick={handleSubmit}
+            disabled={
+              title.trim().length < 1
+              || imgUrl.trim().length < 1
+              || imdbUrl.trim().length < 1
+              || imdbId.trim().length < 1
+            }
```

Comment:
better to create separate variable
```suggestion
            disabled={isDisabled}
```